Mathlib/NumberTheory/PrimeCounting.lean

```diff
@@ -102,7 +102,30 @@ theorem primeCounting'_add_le {a k : ℕ} (h0 : 0 < a) (h1 : a < k) (n : ℕ) :
 theorem zeroth_prime_eq_two : nth Prime 0 = 2 := nth_count prime_two
 
 /-- The `n`th prime is greater or equal to `n + 2`. -/
-lemma add_two_le_nth_prime (n : ℕ) : n + 2 ≤ nth Prime n :=
+theorem add_two_le_nth_prime (n : ℕ) : n + 2 ≤ nth Prime n :=
   zeroth_prime_eq_two ▸ (nth_strictMono infinite_setOf_prime).add_le_nat n 0
 
+theorem surjective_primeCounting' : Function.Surjective π' := by
+   intro n
+   use nth Prime n
+   exact Nat.count_nth_of_infinite infinite_setOf_prime n
+
+theorem surjective_primeCounting : Function.Surjective π := by
```

Comment:
Do we have that `π (n - 1) = π' n`? This should lead to a much shorter proof using [`Function.Surjective.of_comp`](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Logic/Function/Basic.html#Function.Surjective.of_comp).

Reply:
No, we don't have that yet, but good idea let's add it!